cfme/infrastructure/host.py

```diff
@@ -620,16 +621,17 @@ def find_quadicon(host, do_not_navigate=False):
         raise HostNotFound("Host '{}' not found in UI!".format(host))
 
 
-def navigate_and_select_all_hosts(host_names, provider_crud=None):
+def navigate_and_select_all_hosts(host_names, provider):
```

Comment:
**Required**
This was an optional parameter, and you kept the logic that deals with it being empty. it should remain an optional parameter. `provider=None`